spec/jobs/rapid_ready_for_decision/disability_compensation_job_spec.rb

```diff
@@ -119,18 +131,22 @@
 
     context 'when the user uuid is not associated with an Account AND the edipi auth header is blank' do
       let(:submission_without_account_or_edpid) do
-        create(:form526_submission,
+        auth_headers.delete('va_eauth_dodedipnid')
+
+        create(:form526_submission, :with_uploads,
                user_uuid: 'nonsense',
-               auth_headers_json: auth_headers.delete('va_eauth_dodedipnid').to_json,
+               auth_headers_json: auth_headers.to_json,
                saved_claim_id: saved_claim.id,
                submitted_claim_id: '600130094')
       end
 
       it 'raises an error' do
-        Sidekiq::Testing.inline! do
-          expect do
-            RapidReadyForDecision::DisabilityCompensationJob.perform_async(submission_without_account_or_edpid.id)
-          end.to raise_error RapidReadyForDecision::DisabilityCompensationJob::AccountNotFoundError
```

Comment:
What do you think about adding this to ensure that the auth_header is what you expect:
```ruby
expect(submission_without_account_or_edpid.auth_headers['va_eauth_dodedipnid']).to be_blank
```

Reply:
Sure. I do plan on refactoring this whole spec file however in a follow-up PR. But that should be a quick enough addition.